csharp-read-write-sheet/Program.cs

```diff
@@ -19,12 +19,10 @@ class Program
 
         static void Main(string[] args)
         {
-            // Get API access token from App.config file or environment
-            string accessToken = ConfigurationManager.AppSettings["AccessToken"];
+            // Get API access token from environment variable SMARTSHEET_ACCESS_TOKEN
```

Comment:
Leave this comment but remove the rest of the code here?

Reply:
Let's change the comment to 
```//The builder will use the access token stored in the environment variable SMARTSHEET_ACCESS_TOKEN```